src/utility/HackerApplicationContext.js

```diff
@@ -122,10 +121,6 @@ export function HackerApplicationProvider({ children }) {
   }) => {
     const mergedApp = {
       ...application,
-      vaccineInfo: {
-        ...application.vaccineInfo,
-        ...vaccineInfo,
-      },
```

Comment:
If an application was started with the old version of portal and finished on the new version, it won't have `vaccineInfo` field when submitted

Reply:
In this case, I think the vaccineInfo field will remain unchanged - correct me if I'm wrong though

Reply:
Oh yes you're right. Thanks for the correction